lib/host/windows/base_view.cpp

```diff
@@ -252,7 +250,7 @@ namespace cycfi { namespace elements
          return {
             down,
             info->click_count,
-            mouse_button::left,
+            which,
```

Comment:
i haven't checked, but is this the same?

Reply:
No, this behaviour is different. Previously it was always returning `mouse_button::left`, now it returns `which` which depends on the switch. Before this change, the variable `which` was unused (so the switch was dead code).